gradle/plugins/src/main/kotlin/com/saveourtool/template/build/mysql-local-run-configuration.gradle.kts

```diff
@@ -2,20 +2,33 @@ package com.saveourtool.template.build
 
 import org.gradle.api.provider.Property
 import org.gradle.kotlin.dsl.create
-import org.intellij.lang.annotations.Language
 
 interface MysqlLocalRunExtension {
     val rootPassword: Property<String>
     val user: Property<String>
     val password: Property<String>
-    val startupPath: Property<String>
+    val databaseName: Property<String>
+    val liquibaseChangelogPath: RegularFileProperty
+    val liquibaseChangelogDir: DirectoryProperty
+    val contexts: ListProperty<String>
 }
 
 val extension: MysqlLocalRunExtension = extensions.create("mysqlLocalRun")
 
-afterEvaluate {
-    val rootPassword: String = extension.password.getOrElse("123")
+val rootPassword: String by lazy {
+    extension.password.getOrElse("123")
+}
+val user: String by lazy {
+    extension.user.getOrElse("root")
+}
+val password: String by lazy {
+    extension.password.getOrElse(rootPassword)
+}
+val databaseName: String by lazy {
+    extension.databaseName.getOrElse("test")
+}
 
+afterEvaluate {
```

Comment:
need tor remove `afterEvaluate` and calculate `dockerComposeContent` as lazy and only inside the task

Reply:
removed